src/pages/Products/ProductList/ProductList.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import './ProductList.scss';
+import OneProduct from '../OneProduct/OneProduct';
+import '../../../config.js';
+
+class ProductList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      products: [],
+      viewProduct: [],
+      moreView: 4,
+      moreButtonHidden: false,
+      selectedSort: 1,
+    };
+  }
+
+  componentDidMount() {
+    this.getData();
+  }
+
+  moreProduct = () => {
+    const { products, viewProduct, moreView } = this.state;
+    console.log(
+      '전체상품 : ',
+      products.length,
+      '현재상품 :',
+      viewProduct.length,
+      '남은 상품 :',
+      products.length - viewProduct.length
+    );
+    let remainProduct = [];
+    let count = 0;
+    let moreViewProduct = [];
+    if (products.length - viewProduct.length > 4) {
+      for (let i = viewProduct.length; i < viewProduct.length + moreView; i++) {
+        remainProduct[count] = products[i];
+        count++;
```

Comment:
- push 함수를 이용하면 count 값을 추적하면서 관리할 필요가 없습니다.
```suggestion
        remainProduct.push(products[i]);
```

Reply:
감사합니다!